recipe_scrapers/settings/default.py

```diff
@@ -0,0 +1,66 @@
+from recipe_scrapers.plugins import (  # SchemaOrgPrioriotyPlugin,; Bcp47ValidatePlugin,
+    ExceptionHandlingPlugin,
+    HTMLTagStripperPlugin,
+    NormalizeStringPlugin,
+    OpenGraphImageFetchPlugin,
+    SchemaOrgFillPlugin,
+)
+
+# Plugins to be attached.
+# The upper most plugin is the "outer most" executed.
+# Check recipe_scrapers.settings.template.py for ways to extend.
+PLUGINS = (
+    ExceptionHandlingPlugin,
+    HTMLTagStripperPlugin,
+    NormalizeStringPlugin,
+    OpenGraphImageFetchPlugin,
+    SchemaOrgFillPlugin,
+    # Bcp47ValidatePlugin,
+    # SchemaOrgPrioriotyPlugin,
```

Comment:
Super small detail, but there's a typo in `SchemaOrgPrioriotyPlugin` here - just mentioning it so that any future search-and-replace type operations catch everything correctly.

(and so that the v13 release is as clean as possible :))

Reply:
thanks!